cmds/exp/acpigrep/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// acpigrep greps a stream for tables with four-byte character signatures
```

Comment:
this description mixes and matches the man-page style with Go style, can we pick one or the other?

Reply:
good point, will fix.

Reply:
well I tried ...